dashboard/pages/correlation/callbacks.py

```diff
@@ -111,6 +113,27 @@ def on_both_files_uploaded(
     return ICON_OK, False
 
 
+def upload_settings_data(content: str | None, name: str | None) -> dict:
+    """
+    Callback for file upload. It saves the in local storage for other components.
+
+    :param content: base64 encoded file content
+    :param name: filename
+    :return: dict with loaded data
+    """
+    if not content:
+        return no_update
+    loaded_data = load_data_from_json(content, name)
+    color = "success"
+    text = "Settings uploaded successful"
+    if loaded_data == None or not set(["concentration_value", "volume_value"]).issubset(
+        loaded_data.keys()
+    ):
+        color = "danger"
+        text = "Settings not uploaded"
```

Comment:
Maybe sth like `Invalid settings uploaded: the file should contain {settings keys} keys.`